packages/modules/src/SessionKeyManagerModule.ts

```diff
@@ -66,11 +65,11 @@ export class SessionKeyManagerModule extends BaseValidationModule {
     instance.nodeClient = new NodeClient({
       txServiceUrl: moduleConfig.nodeClientUrl ?? NODE_CLIENT_URL,
     });
-
-    if (!moduleConfig.storageType || moduleConfig.storageType === StorageType.LOCAL_STORAGE) {
```

Comment:
can we not remove this storageType please

instead I suggest

if (moduleConfig.sessionStorageClient) {
      instance.sessionStorageClient = moduleConfig.sessionStorageClient;
    } else {
    
    // switch case here on moduleConfig.storageType if provided
    // if case if LOCAL_STORAGE then create SessionLocalStorage
    // throw in default case
    
}